lib/internal/main/run.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+/* eslint-disable node-core/prefer-primordials */
+
+// There is no need to add primordials to this file.
+// `run.js` is a script only executed when `node run <script>` is called.
+const {
+  prepareMainThreadExecution,
+  markBootstrapComplete,
+} = require('internal/process/pre_execution');
+const { getPackageJSONScripts } = internalBinding('modules');
+const { execSync } = require('child_process');
+const { resolve, delimiter } = require('path');
+const { escapeShell } = require('internal/shell');
+
+prepareMainThreadExecution(false, false);
+
+markBootstrapComplete();
+
+// TODO(@anonrig): Search for all package.json's until root folder.
+const json_string = getPackageJSONScripts();
+
+// Check if package.json exists and is parseable
+if (json_string === undefined) {
+  process.exit(1);
```

Comment:
We should probably print an error message before exiting.

Reply:
The error message is thrown from C++